kubeconfig.tmp

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
Are those files included by mistake here or we need in some cases a dummy kubeconfig file for testing?

Reply:
Probably should be deleted